opam

```diff
@@ -25,7 +25,7 @@ depends: [
   "ppx_sexp_conv"
   "uri"
   "re"
-  "mtime" {< "1.0.0"}
+  "mtime" {>= "1.0.0"}
```

Comment:
Let's be proactive and add `& < "2.0.0"`

Reply:
Done.